epsagon/trace.py

```diff
@@ -1087,7 +1095,7 @@ def _send_traces(self):
             ))
         finally:
             if self.debug:
-                pprint.pprint(self.to_dict())
+                print('trace:', trace)
```

Comment:
This fixes the trace print to be in one line and readable, instead of many lines.